README.md

```diff
@@ -0,0 +1,9 @@
+# Sparkplug
+
+[![Build Status](https://www.travis-ci.org/freshbooks/sparkplug.svg?branch=master)](https://www.travis-ci.org/freshbooks/sparkplug)
+
+## Testing
+run tests with `nosetests`
+
+## Releasing
+To create a new release commit run `./bump.sh`. When your branch is merged with master, a new release will be automatically run.
```

Comment:
You can tell travis to automate this entire process. Just add a new stage, that only cuts a release when merged into master.

```
stages:
  - name: release
    if: branch = master
    script: 
     - pip install bumpversion
     - bumpversion patch
```

Reply:
The issue I've been thinking about with that is: `bumpversion` makes a commit, which should then kick off another build, and then it'll just recursively create builds forever

Reply:
might be able to get around this by checking for revisions since the last version change, or checking that the current commit isn't a bumpversion commit

Reply:
seems like something that should be pretty standard 🤔

Reply:
There's also a python package you can check out: https://pypi.org/project/travis-bump-version/